app/src/components/Scheduling/Header.js

```diff
@@ -41,15 +27,14 @@ export default function Header({ textOptions }) {
           marginBottom: '0.5rem',
         }}
         color='primary'>
-        Training
+        {activity}
       </Typography>
       <Typography
-        variant='body2'
+        variant='subtitle1'
         style={{
-          fontSize: '14px',
           marginBottom: '1rem',
         }}>
-        {getActionText()}
+        {children}
```

Comment:
What should `children` be? When I log it to the console it's `undefined`. I also have doubts that children is the correct var name for this instance. Children is used more often a component is passed as props.

Reply:
Ahh I see. I definitely named it wrong then. `children` renders the action text depending on the page. These are seen in the calendar view and choose time availability pages. 

I'm thinking the content in `children` can be moved to the page itself (CalendarView) and placed right below the `<Header />`. Then remove the `children` prop from `<Header />`.

This is how I would restructure the header in `CalendarView.js` for the above solution:

```
<Header
  title="Schedule with Tracy"
  description="Select from Tracy's availability for"
  activity="Training"
>
  {buttonDisabled
    ? 'Choose a date'
    : 'Date selected'
  }
</Header>
```

To:

```
<Header
  title="Schedule with Tracy"
  description="Select from Tracy's availability for"
  activity="Training"
/>
<h5>
  {buttonDisabled
    ? 'Choose a date'
    : 'Date selected'
  }
</h5>
```

A different soltuion I can think of is renaming `children` to `action`.

I'm currently leaning more toward the first solution to avoid having `children` be undefined when it is not being used in other pages such as the `<ContactReview />` page.

In this screenshot, it's the very bottom text that says "3 out of 3 days chosen".
![Screen Shot 2021-05-04 at 8 50 41 AM](https://user-images.githubusercontent.com/17840013/117031890-de166600-acb5-11eb-9b85-0bd031b1204e.png)

Reply:
Just pushed the first solution!